catalog-rest-service/src/main/resources/ui/src/components/common/CronEditor/CronEditor.jsx

```diff
@@ -0,0 +1,556 @@
+/*
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements. See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License. You may obtain a copy of the License at
+
+  * http://www.apache.org/licenses/LICENSE-2.0
+
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+/* eslint-disable */
+
+import React, { useState } from 'react';
+import {
+  combinations,
+  getDayCron,
+  getDayOptions,
+  getHourCron,
+  getHourOptions,
+  getMinuteCron,
+  getMinuteOptions,
+  getMonthCron,
+  getMonthDaysOptions,
+  getMonthOptions,
+  getPeriodOptions,
+  getWeekCron,
+  getYearCron,
+  toDisplay,
+} from './CronEditor.constant';
+import './CronEditor.css';
```

Comment:
As discussed this should be from styles folder, and imported into `index.js` there